index.html

```diff
@@ -134,5 +137,18 @@
             </div>
         </article>
 
+        <section class="section-3">
+            <blockquote class="blockquote blockquote-bordered quote-mark">
+                <p class="blockquote_text">
+                    Somewhere, something incredible is waiting to be known.
+                </p>
+                <p class="blockquote_text blockquote_author">Carl Sagan</p>
+            </blockquote>
+
+            <div class="quote-img-container">
+                <img class="quote-img" src="./Images/quote-img.jpg" alt="an image of a person standing next to a telescope facing upward and looking up at the stars in the night sky.">
```

Comment:
An image was obtained as required by #5 to place beside the quote in the webpage